pkg/config/config.go

```diff
@@ -201,6 +216,18 @@ func (vcConfig *VCConfig) validate() error {
 	return nil
 }
 
+func (wcpConfig *WCPConfig) validate() error {
+	enableVPC := wcpConfig.EnableWCPVPCNetwork
+	if enableVPC {
```

Comment:
if VPC is not enabled, do we need to pass the validation here?

Reply:
no, if VPC is not enabled, no validation needed.